testsuite/features/step_definitions/retail_steps.rb

```diff
@@ -63,6 +63,24 @@ def compute_image_name
   end
 end
 
+def compute_image_name_version
```

Comment:
I would like to refactor these two methods `compute_image_name_version` and `compute_image_name` for the sake of maintenability.
An idea could be to return `name, version` in a method, so then from wherever we call them we can build the string using only first variable `name` or both "#{name}:#{version}"